docs/Data/Bioinformatics_Pipelines/CNV_Pipeline.md

```diff
@@ -9,7 +9,7 @@ The copy number variation (CNV) pipeline uses Affymetrix SNP 6.0 array data to i
 
 A metadata preprocessing step is used to convert the GRCh37 (hg19) probe set coordinates to the newer GRCh38 (hg38) genome build coordinates. A minimum quality control step to verify that reference bases are consistent across two genome builds is used to filter out low quality liftover probe sets.
 
-The [Copy Number Liftover Workflow](/Data_Dictionary/viewer/#?view=table-definition-view&id=copy_number_liftover_workflow) uses the TCGA level 2 tangent.copynumber files described above. These files were generated by first normalizing array intensity values, estimating raw copy number, and performing tangent normalization, which subtracts variation that is found in a set of normal samples. Original array intensity values (TCGA level 1) are available in the [GDC Legacy Archive](https://portal.gdc.cancer.gov/legacy-archive/) under the "Data Format: CEL" and "Platform: Affymetrix SNP 6.0" filters.
```

Comment:
These are available in the active portal now.  You should put the sentence back, but remove the reference to TCGA level 1.  Also, verify that the filters are still valid.

Reply:
Change "active Archive" to "Data Portal"  and this will be ready.